emacs/redtt.el

```diff
@@ -0,0 +1,187 @@
+;;; redtt.el --- Major mode for editing redtt proofs and interacting with redtt  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2016  Jonathan Sterling
+;; Copyright (C) 2017  Jonathan Sterling, Favonia
```

Comment:
Maybe we should update this to 2018. lol

Reply:
lol